webpack.config.js

```diff
@@ -0,0 +1,43 @@
+var webpack = require('webpack');
+
+var isDev = process.env.NODE_ENV !== 'production';
+var plugins = [];
+
+if(!isDev) {
+  plugins.push(new webpack.optimize.UglifyJsPlugin({
+      compress: isDev,
+      mangle: false
+    })
+  );
+}
+
+module.exports = {
+  entry: "./src/js/index.js",
+  devtool: isDev ? 'cheap-module-source-map' : false,
+  output: {
+    path: __dirname,
+    filename: "compiled.js"
+  },
+  watch: isDev,
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: {
+          presets: ['es2015']
```

Comment:
https://github.com/cloudflare/cloudflare-plugin-frontend/blob/master/.babelrc

I don't think we need `presets ` we already have .babelrc. I'm not sure if the whole loader field is necessary.

Alternatively, we can delete .babelrc, and `babelrc: false` to webpack config.